content/guides/cli/scripts/manage-groups-collaborations.md

```diff
@@ -0,0 +1,193 @@
+---
+rank: 7
+related_endpoints: []
+related_guides:
+  - authentication/oauth2
+  - cli/scripts/provision-users-folders
+  - cli/scripts/deprovision-users
+  - cli/scripts/user-zones-mass-update
+  - cli/scripts/report-inactive-users
+related_pages:
+  - sdks-and-tools
+required_guides:
+  - authentication/oauth2/oauth2-setup
+related_resources: []
+
+
+---
+# Manage groups and collaborations
+
+<!-- markdownlint-disable line-length -->
+
+## Script structure
+
+This script uses Box CLI to create or update groups, add users to them, and finally to create collaborations between groups and folders.
+The script consists of the following two parts. You can run them both or select one of them.
```

Comment:
Suggest to define what two parts or just say "The script consists of the two parts"

Reply:
Sure, I will perhaps write that the parts are explained in the next sections.